pilot/pkg/config/kube/gateway/testdata/eastwest-labelport.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: Gateway
+metadata:
+  name: eastwestgateway
+  namespace: istio-system
+  labels:
+    topology.istio.io/network: "network-1"
```

Comment:
Istiod in the cluster is supposed to know the network id it is operating on and to add it to all pods - i.e. the controller or template should already handle this automatically.  

No need for the user to customize a Gateaway per network and remember to copy the same name used at istiod install. 

How about having a new label to indicate this gateway is designated for E/W ? 

I guess you are setting it here for convenience in the code ( to get the network from watching the gateway ), but it becomes a bit harder for the user. 

May not be a huge deal and we may fix it when we move to hbone...

Reply:
The intent is that I am writing a Gateway resource to represent a _remote_ gateway. Similar to the WorkloadEntry proposal, this allows discovery of remote resources without watching a remote API server.

Reply:
> we can have multiple networks in one cluster.

Are you referring to the case of multiple nics?